src/index.ts

```diff
@@ -23,7 +27,7 @@ export {
   findRouterContext,
 } from './common/utils';
 
-export type {
+export {
```

Comment:
@rohan-deshpande the reason we don't need `export type` here, is because in `src/common/types` we are exporting `LocationShape` as a type now. 

Previously TS was unable to determine if `LocationShape` was a type or an actual JS property so the `export type` here was covering up the actual issue by forcing TS to treat it as a type.